test/index.js

```diff
@@ -379,6 +381,18 @@ describe('Memory', function () {
         done();
     });
 
+    it('cleans up timers when stopped', function (done) {
+
+        var start = Date.now();
+        var child = ChildProcess.fork(Path.join(__dirname, 'fixtures', 'timers'));
+        child.on('exit', function () {
+
+            var stop = Date.now();
+            expect(stop - start).to.be.lessThan(500);
```

Comment:
Agreed. I tried to avoid this the first time around, but didn't come up with anything. I have an idea that might work though -- will give it a try. Thanks for the extra push :-)